libs/langchain/langchain/smith/evaluation/progress.py

```diff
@@ -0,0 +1,25 @@
+"""A simple progress bar for the console."""
+from threading import Lock
+
+
+class ProgressBar:
```

Comment:
Btw we have tqdm as a transitive dependency already courtesy of openai sdk

Reply:
Oh that's even nicer. I almost never run into scenarios where people don't have tqdm so i'll just use that, just didn't want to add another requirement

Reply:
Hm actually just gonna keep the print method since it works and pbar kept not erasing